src/main/java/uk/ac/ebi/embl/converter/gff3toff/GFF3Mapper.java

```diff
@@ -89,12 +90,30 @@ private Feature mapGFF3Feature(GFF3Feature gff3Feature) {
         Map<String, String> attributes = gff3Feature.getAttributes();
         Collection<Qualifier> qualifiers = mapGFF3Attributes(attributes);
```

Comment:
Suggestion to directly call `mapGFF3Attributes(attributes);` instead of creating a variable
`ffFeature.addQualifiers(mapGFF3Attributes(attributes));`

Reply:
Sorry, maybe my comment was not clear enough. 
Suggestion was to remove:
`Collection<Qualifier> qualifiers = mapGFF3Attributes(attributes);`

and call mapGFF3Attributes(attributes); directly as below

` ffFeature.addQualifiers(mapGFF3Attributes(attributes););`